handlers/handlers.http

```diff
@@ -0,0 +1,15 @@
+@baseurl = http://localhost:8080/api
```

Comment:
Nice addition! Remember that your README could benefit by having instructions on how files like these work and what you need to actually make use of them.